examples/jsm/renderers/common/Renderer.js

```diff
@@ -1086,9 +1086,9 @@ class Renderer {
 	}
 
 
-	readRenderTargetPixelsAsync( renderTarget, x, y, width, height ) {
+	readRenderTargetPixelsAsync( renderTarget, x, y, width, height, index = 0 ) {
 
-		return this.backend.copyTextureToBuffer( renderTarget.texture, x, y, width, height );
+		return this.backend.copyTextureToBuffer( renderTarget.textures[ index ], x, y, width, height );
```

Comment:
Does this code...actually work? At least for the WebGL backend won't work until `gl.readBuffer()` is used in `copyTextureToBuffer()`.

I'm not sure about required workflow in WebGPU. Do you mind demonstrating the updated method with an example? E.g. you could use the code from https://threejs.org/examples/webgpu_multiple_rendertargets but instead of rendering to the screen, you extract pixel values via `readRenderTargetPixelsAsync()`.